qa/integration/services/helpers.sh

```diff
@@ -21,6 +29,12 @@ wait_for_port() {
     done
     # just in case, one more time
     nc -z localhost $port
+
+}
+
+wait_for_port_sleep() {
+  echo "nc not installed on this machine. Sleeping for 10 seconds"
```

Comment:
if we know ruby is installed (we know jruby is, at least), maybe we can use `ruby -rsocket -e 'TCPSocket.new(ENV["host"], ENV["port"]) rescue exit(1)'`? the rescue just makes it less noisy, but can be removed